spec/features/photos_features_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+feature 'photos' do
+  context 'no photos have been added' do
+    scenario 'should display a prompt to add a picture' do
+      visit '/photos'
+      expect(page).to have_content 'no photos yet'
+      expect(page).to have_link 'Add Photo'
+    end
+  end
+
+  context 'creating photos' do
+
+    scenario 'prompts user to fill out a form, then displays the new photo' do
+      sign_in
+      add_picture
+      expect(page).not_to have_content 'no photos yet'
+      expect(page).to have_content 'test'
+      expect(page).to have_selector "img"
+    end
+  end
+  context 'viewing photos' do
+    let!(:test_pic){Photo.create(name:'testing_picture')}
```

Comment:
Space missing after colon.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.